include/aos/cni/cni.hpp

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_CNI_HPP_
+#define AOS_CNI_HPP_
+
+#include "aos/common/tools/array.hpp"
+#include "aos/common/tools/error.hpp"
+#include "aos/common/tools/string.hpp"
+#include "aos/common/tools/uuid.hpp"
+#include "aos/common/types.hpp"
+
+#include "config.hpp"
+
+namespace aos {
+namespace cni {
+
+/**
+ * Max CNI version length.
+ */
+constexpr auto cVersionLen = AOS_CONFIG_CNI_VERSION_LEN;
+
+/**
+ * Max CNI plugin name length.
+ */
+constexpr auto cNameLen = AOS_CONFIG_CNI_PLUGIN_NAME_LEN;
+
+/**
+ * Max CNI plugin type length.
+ */
+constexpr auto cPluginTypeLen = AOS_CONFIG_CNI_PLUGIN_TYPE_LEN;
+
+/**
+ * Max CNI router number.
+ */
+constexpr auto cMaxNumRouters = AOS_CONFIG_CNI_MAX_NUM_ROUTERS;
+
+/**
+ * Max CNI number of interfaces.
+ */
+constexpr auto cMaxNumInterfaces = AOS_CONFIG_CNI_MAX_NUM_INTERFACES;
+
+/**
+ * Max CNI number of IPs.
+ */
+constexpr auto cMaxNumIPs = AOS_CONFIG_CNI_MAX_NUM_IPS;
+
+/**
+ * Max CNI number of runtime config arguments.
+ */
+constexpr auto cMaxNumRuntimeConfigArgs = AOS_CONFIG_CNI_MAX_NUM_RUNTIME_CONFIG_ARGS;
+
+/**
+ * Max CNI runtime config argument length.
+ */
+constexpr auto cRuntimeConfigArgLen = AOS_CONFIG_CNI_RUNTIME_CONFIG_ARG_LEN;
+
+/**
+ * CNI version.
+ */
+constexpr auto cVersion = AOS_CONFIG_CNI_VERSION;
+
+/**
+ * CNI binaries path.
+ */
+constexpr auto cCNIBinariesPath = AOS_CONFIG_CNI_BINARIES_PATH;
```

Comment:
I prefer to leave the prefix CNI, to reduce possible coalition in name, and also it would be better to understand this config is part of CNI configuration

Reply:
Done